### PR TITLE
Feature kumaresan

### PR DESCRIPTION
**Pull request by Kumaresan227**
update the subject

### CODE REVIEW

File: _slides/03-Kumaresan227.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+New Datas
```
Comment (renan-alm) [inline _slides/03-Kumaresan227.md:6]:
```suggestion
New Data!!!
```

File: _slides/03-Kumaresan227.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (renan-alm) [inline _slides/03-Kumaresan227.md:3]:
```suggestion
title: "This is Kuma octocat!"
```

**Review by renan-alm — APPROVED** [review]

**Review by Isenthal — COMMENTED** [review]
test

File: _slides/03-Kumaresan227.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
```
Comment (Isenthal) [inline _slides/03-Kumaresan227.md:3]: test

**Review by Isenthal — APPROVED** [review]

File: _slides/03-Kumaresan227.md (hunk 3 above)
Comment (Kumaresan227) [inline _slides/03-Kumaresan227.md:3]: comment

**Review by Kumaresan227 — COMMENTED** [review]
approved

**Pull request merged**